osu.Game/Rulesets/Objects/HitObject.cs

```diff
@@ -144,9 +144,10 @@ protected virtual void CreateNestedHitObjects()
 
         /// <summary>
         /// Creates the <see cref="Judgement"/> that represents the scoring information for this <see cref="HitObject"/>.
-        /// May be null.
+        /// Used to decide on drawable object lifetimes.
```

Comment:
```suggestion
```

Really unnecessary. It's used to score the hitobject primarily, lifetime only occurs as a consequence of doing so.